extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/IcebergCatalogRangerSparkExtensionSuite.scala

```diff
@@ -235,13 +235,15 @@ class IcebergCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite
       s" on [$namespace1/$table1]"))
   }
 
-  test("CALL RewriteDataFilesProcedure") {
-    val tableName = "call_command_table"
-    val table = s"$catalogV2.$namespace1.$tableName"
-    val rewriteCommand1 = s"CALL $catalogV2.system.rewrite_data_files " +
+  test("CALL RewriteProcedure") {
```

Comment:
We'd better use RewriteDataFilesProcedure, refering to `org.apache.iceberg.spark.procedures.RewriteDataFilesProcedure` from Iceberg Spark plugin.